include/swift/AST/Module.h

```diff
@@ -871,6 +871,28 @@ class ModuleDecl
   /// \sa getImportedModules
   using ImportFilter = OptionSet<ImportFilterKind>;
 
+  /// Returns an \c ImportFilter with all elements of \c ImportFilterKind.
+  constexpr static ImportFilter getImportFilterAll() {
```

Comment:
Should we just have separate `getImportedModules` functions for all/local?

Reply:
That would work too, I like the sets as we can union sets of filters. There's a few use sites that do that already, but I don't think I've updated them to use these sets.